node/tests/commons/persistent-options.spec.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs-extra');
+const persistentOptions = require('../../src/commons/persistent-options');
+const { OPTIONS } = require('../../src/commons/constants');
+
+jest.mock('fs-extra');
+
+
+describe('persisted options', () => {
+    const mockOptions = {
+        [OPTIONS.API_KEY]: 'key0',
+        [OPTIONS.API_SECRET]: 'secret0'
+    };
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    })
+
+    describe('read', () => {
+        describe('when config file exists', () => {
+            beforeEach(() => {
+                jest.spyOn(fs, 'readFileSync').mockReturnValue(JSON.stringify(mockOptions));
+                jest.spyOn(fs, 'existsSync').mockReturnValue(true);
+            })
```

Comment:
```suggestion
            });
```

no lint?

Reply:
I'll refactor lint + prettifier + ts when I'll work on the publish to npm task